ansible/roles/nginx/templates/server.config.j2

```diff
@@ -122,6 +122,14 @@ location /resources {
 
 # Always enforce same origin policy
 add_header X-Frame-Options SAMEORIGIN;
+# XSS-Protection - Enables XSS filtering. Rather than sanitizing the page, the browser will prevent rendering of the page if an attack is detected.
+add_header X-XSS-Protection "1; mode=block";
+# Content security policies allowing content to be loaded from specified addresses
+add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' platform.twitter.com syndication.twitter.com cdn.syndication.twimg.com localhost-avoindata-fi.disqus.com https://disqus.com *.disquscdn.com; img-src 'self' data: *.twitter.com *.twimg.com *.disqus.com *.disquscdn.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://platform.twitter.com https://ton.twimg.com *.disquscdn.com; font-src 'self' https://themes.googleusercontent.com; frame-src syndication.twitter.com https://platform.twitter.com https://disqus.com *.disqus.com; object-src 'none'; connect-src 'self' https://links.services.disqus.com wss://realtime.services.disqus.com";
```

Comment:
localhost-avoindata-fi.disqus.com needs to be changed to enviroment specific one or to the wildcard.